packages/ui5-application-inquirer/README.md

```diff
@@ -0,0 +1,79 @@
+# @sap-ux/ui5-application-inquirer
+
+Provides Inquirer based prompting to allow input and validation of data required to generate a UI5 library
+
+## Installation
+Npm
+`npm install --save @sap-ux/ui5-application-inquirer`
+
+Yarn
+`yarn add @sap-ux/ui5-application-inquirer`
+
+Pnpm
+`pnpm add @sap-ux/ui5-application-inquirer`
+
+## Explainer
+
+Prompts may be retrieved using `getPrompts` and then executed in another prompting module that supports `inquirer` type prompts. However, it is recommended to call `prompt` so that unanswered prompts can be assigned defaults if provided via prompt options before returning, otherwise this needs to be taken care of by the callers prompting functionality. Note that hidden prompts, for example those hidden behind the advanced option, may not have be executed, depending on the users selection of `showAdvanced`. In addition, previous answers need to be provided to default functions, and this is taklen care of when calling `prompt` but would need to be handled by the consumers prompting framework if using the prompts returned by `getPrompts` directly.
```

Comment:
Couple of bits need fixing here `may not have be executed` & `is taklen care of`